tests/Rector/StaticCall/RouteActionCallableRector/Fixture/fixture.php.inc

```diff
@@ -15,6 +15,7 @@ Route::match(['GET', 'POST'], '/users', 'SomeController@index');
 Route::fallback('SomeController@index');
 Route::options('/users', 'SomeController@index');
 Route::middleware([])->options('/users', 'SomeController@index');
+Route::get('/users', ['as' => 'users.index', 'uses' => 'SomeController@index']);
```

Comment:
what if the structure flipped, `uses` key first, `as` key last, still applied?

Reply:
Still fails with the keys flipped

Reply:
Could you add 1 more line for flipped usage to ensure all covered? Thanks

Reply:
Added new commit for flipped usage